include/picolibc-freertos.h

```diff
@@ -43,22 +43,43 @@
 
 #define configTLS_BLOCK_TYPE               void *
 
+#define configTLS_SIZE                     ( ( portPOINTER_SIZE_TYPE ) _tls_size() )
```

Comment:
As per the FreeRTOS naming convention, the macros that start with `config` are user configurations and can be defined in FreeRTOSConfig.h. How about we update the following macros to start with `picolibc` instead?

* picolibcTLS_SIZE
* picolibcSTACK_ALIGNMENT_MASK
* picolibcTLS_ALIGNMENT_MASK

Reply:
thanks for the good suggestion.